<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imc</class>
 <widget class="QMainWindow" name="imc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>441</width>
         <height>281</height>
        </rect>
       </property>
       <property name="title">
        <string>Datos </string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Arial Narrow</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Ingrese Fecha:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="intFecha">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2022</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yyyy</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>Arial Narrow</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Ingrese Altura:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="intEstatura">
             <property name="maximum">
              <number>300</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>Arial Narrow</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Ingrese Peso:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="intPeso">
             <property name="maximum">
              <number>400</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Arial Narrow</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Actual :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="outPA">
             <property name="text">
              <string>0.0 kg</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Arial Narrow</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Peso maximo: </string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="outPM">
             <property name="text">
              <string>0.0 kg</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Arial Narrow</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Peso minimo :</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="outPMIN">
             <property name="text">
              <string>0.0 kg</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="btCalcular">
             <property name="text">
              <string>CALCULAR</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>441</width>
         <height>221</height>
        </rect>
       </property>
       <property name="title">
        <string>Total</string>
       </property>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>413</width>
          <height>171</height>
         </rect>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>IMC (kg/m) : </string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>347</width>
           <height>61</height>
          </rect>
         </property>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(124, 242, 255);</string>
          </property>
         </widget>
         <widget class="Line" name="outLinea">
          <property name="geometry">
           <rect>
            <x>92</x>
            <y>0</y>
            <width>4</width>
            <height>58</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>58</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>4</width>
            <height>46</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QWidget" name="widget_8" native="true">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 255, 38);</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_9" native="true">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>10</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 143, 15);</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_10" native="true">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>10</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(162, 0, 3);</string>
          </property>
         </widget>
         <zorder>widget_10</zorder>
         <zorder>widget_7</zorder>
         <zorder>widget_8</zorder>
         <zorder>widget_9</zorder>
         <zorder>outLinea</zorder>
        </widget>
        <widget class="QLabel" name="outTIMC">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>188</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="outimc">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>140</y>
           <width>231</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>441</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Autor</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>UNIVERSIDAD POLITECNICA SALESIANA </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RAINER VERGARA NAZARENO </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>70</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>rvergaran1@est.ups.edu.ec</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>EXAMEN 2DO PARCIAL</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
